src/lib/dai.c

```diff
@@ -43,6 +43,11 @@ __cold static struct dai_group_list *dai_group_list_get(int core_id)
 	if (!group_list) {
 		group_list = rzalloc(SOF_MEM_FLAG_USER,
 				     sizeof(*group_list));
+		if (!group_list) {
+			tr_err(&dai_tr, "dai_group_list_get(): failed to allocate group_list for core %d",
+			       core_id);
```

Comment:
we should ask AI to make a tree-wide PR to remove function names from all prints ;-)

Reply:
> yeah, although we don't yet have the function and line number on the CI logs yet today.

@lgirdwood we do have function names https://sof-ci.01.org/sofpr/PR10088/build13773/devicetest/index.html?model=PTLH_RVP_NOCODEC&testcase=check-playback-3times
```
[  408.856116] <inf> ll_schedule: zephyr_ll_task_schedule_common: task add 0xa0101840 0xa0092524U priority 0 flags 0x0
[  408.856701] <inf> dai_intel_ssp: dai_ssp_get_properties: SSP16: fifo 168420, handshake 2, init delay 0
[  408.856708] <inf> dai_intel_ssp: dai_ssp_early_start: SSP16 RX
[  408.856726] <inf> dai_intel_ssp: dai_ssp_start: SSP16 RX
```